### DISCUSSION TITLE
Why I try to connect Android device, It's always display "Disconnected client 127.0.0.1"?

### DISCUSSION THREAD

**Discussion by wa143825** [pawelsalawa/sqlitestudio#4145] [Q&A]
When I connect my physics device by SqLiteStudio.It's can find my device, but the button of "ok" show grey.And The Android Studio Console print this.

D/SQLiteStudioRemote(25496): New client from 127.0.0.1
D/SQLiteStudioRemote(25496): Disconnected client 127.0.0.1

**Comment by pawelsalawa** [comment]
Are you able to connect to your device with the `adb`? Are you able to run shell on your device? `adb shell`

**Reply by wa143825** [reply]
Yes, everything is fine. The Database Inspector of Android Studio can also manager the Sqlite, It is useable.
But in SqliteStudio, every time I open the Dialog, the Console print this again.
[Dialog](https://i.ibb.co/9n94bXK/php-J2-Fo-Cl.jpg)
[Console](https://i.ibb.co/2N99GXN/php-Wv8-HQU.jpg)

**Reply by pawelsalawa** [reply]
Try with the second connection method:
![image](https://user-images.githubusercontent.com/481846/124355918-e4708200-dc13-11eb-96db-4e21dbd507c6.png)

I'm not sure why it's failing for you. I could only figure it out if I reproduced it locally, but I don't have an android environment configured on my PC right now. And even if I had, it's not certain it would be reproducible for me.
